app/views/my/account.html.erb

```diff
@@ -48,9 +48,9 @@ See doc/COPYRIGHT.rdoc for more details.
             <%= @user.login %>
           </div>
         </div>
-        <div class="form--field"><%= f.text_field :firstname, required: true %></div>
-        <div class="form--field"><%= f.text_field :lastname, required: true %></div>
-        <div class="form--field"><%= f.text_field :mail, required: true %></div>
+        <div class="form--field"><%= f.text_field :firstname, required: true, container_class: '-middle' %></div>
+        <div class="form--field"><%= f.text_field :lastname, required: true, container_class: '-middle' %></div>
+        <div class="form--field"><%= f.text_field :mail, required: true, container_class: '-middle' %></div>
```

Comment:
Looking good :+1: but please have a look at the custom fields being included via partial [down below](https://github.com/Azure7111/openproject/pull/2/files#diff-19275ccfdb0585059d8cdaea4a6cff48R61). To style them you'll probably have to work on `customizable/_field.html.erb`